### PR TITLE
OracleDB Plugin Doc

### PR DESCRIPTION
**Pull request by cpheps**
### Proposed Change
Added documentation for OracleDB Plugin.

We will have to add more extensive documentation to all other plugins and link them to the main readme at some point. This is the first as it is needed by the Field team for customer engagement.

##### Checklist
- [x] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: docs/plugins/oracledb_metrics.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+## Least Privileged User
+
+To collect metrics, a monitoring user requires `SELECT` access to the relevant views. The following sql
+script should create a monitoring user and give it the appropriate permissions if executed by
+a user with sufficient permissions connected to the Oracle DB instance as SYSDBA or SYSOPER.
+
+```sql
+  <pre>
```
Comment (BinaryFissionGames) [inline docs/plugins/oracledb_metrics.md:46]: Is this pre-tag actually needed?

File: docs/plugins/oracledb_metrics.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Below is an example of basic config using `sid` and user/pass.
+
+```yaml
+receivers:
+  plugin:
+    path: ./plugins/oracledb_metrics.yaml
+    parameters:
+        endpoint: oracle_db:1521
+        username: otel
+        username: otel_pass
```
Comment (BinaryFissionGames) [inline docs/plugins/oracledb_metrics.md:35]:
```suggestion
        password: otel_pass
```

**Review by BinaryFissionGames — APPROVED** [review]

File: docs/plugins/oracledb_metrics.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+## Supported Versions
+
+- 12.2
+- 18c
+- 19c
+- 21c
+
+## Configuration Parameters
+
+| Name | Description | Type | Default | Required | Values |
```
Comment (JonathanWamsley) [inline docs/plugins/oracledb_metrics.md:14]: What is the purpose of the Value column if all values are empty?

Comment (cpheps) [reply docs/plugins/oracledb_metrics.md:14]: It's for enum typed parameters. I plan on the documentation tool autogenerating docs like this and tried to make this doc as close to the template I envisioned as possible. That way it won't change significantly once we have a generation tool.

**Pull request merged**